src/test/java/spec/yandexSpec/YandexMailSpecs.java

```diff
@@ -0,0 +1,70 @@
+package spec.yandexSpec;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pages.yandex.HomePage;
+import pages.yandex.LogInPage;
+import pages.yandex.MailPage;
+import pages.yandex.YandexMainPage;
+import projectVeriables.ProjectVariables;
+import spec.BaseSpec;
+
+public class YandexMailSpecs extends BaseSpec {
+    private static final Logger logger = LoggerFactory.getLogger(YandexMailSpecs.class);
+    private HomePage homePage;
+
+    @Test
+    public void verifyYandexHomePage() {
+        homePage = new HomePage();
+        driver = homePage.getDriver();
+
+        Assertions.assertTrue(homePage.verifyHomePage());
+    }
+
+    @Test
+    public void verifyUserCanLogInSuccessfully() {
+        homePage = new HomePage();
+        driver = homePage.getDriver();
```

Comment:
You don't need WebDriver instance in tests, because you encapsulate all manipulations with elements in page objects. Remove